ocs_sample_library_preview/Assets.py

```diff
@@ -0,0 +1,523 @@
+import json
+from typing import List
+
+from .BaseClient import BaseClient
+from .Asset.Data.DataResults import DataResults
+from .Asset.Resolved.ResolvedAsset import ResolvedAsset
+from .Asset.Status.StatusData import StatusData
+from .Asset.Asset import Asset
+from .Asset.AssetType import AssetType
+
+
+class Assets(object):
+    """
+    Client for interacting with OCS Assets
+    """
+
+    def __init__(self, client: BaseClient):
+        """
+        Initializes the Assets client
+        :param client: This is the base client that is used to make REST calls
+        """
+        self.__base_client = client
+        self.__setPathAndQueryTemplates()
+
+    def getAssetById(self, namespace_id: str, asset_id: str) -> Asset:
+        """
+        Returns the specified asset
+        :param namespace_id: The namespace identifier
+        :param asset_id: The asset identifier
+        """
+        if namespace_id is None:
+            raise TypeError
```

Comment:
should the TypeErrors throughout this file have comments, such as raise TypeError('namespace_id cannot be None')?

Reply:
I followed the existing pattern from other services (Streams, DataViews), which is not to add a comment. I don't necessarily disagree but I think that might be outside the scope of this PR.